src/riak_kv_compile_tab.erl

```diff
@@ -165,6 +166,16 @@ get_ddl(BucketType) when is_binary(BucketType) ->
             notfound
     end.
 
+%%
+-spec get_all_compiled_ddls() -> term() | notfound.
+get_all_compiled_ddls() ->
+    case dets:match(?TABLE, {'_','_','$1','_','compiled'}) of
```

Comment:
No, but all versions of the DDL will be stored in this dets.  So that on downgrade the correct version can be picked out, because it may not be possible to have a downgrade transform for ddl_v10 when we only know about version ddl_v8 for example.